### ISSUE TITLE
[Suggestion] Changing the name of Sigil of Haste

### ISSUE THREAD

**Issue by VincentLongiug** [WayofTime/BloodMagic#1626]
In BM, you have the Fast Miner Array and the **Sigil of the Fast Miner** granting the Haste Buff, which increases Mining speed.

But the **Sigil of Haste** gives a Boost Buff, sort of mix between Speed II, Jump Boost II and Step Assist, without any effect on Mining speed.

So, to avoid any confusion, I would like to suggest changing the name of Sigil of Haste into something a bit more fancy (and relevant ?), like **Sigil of the Horse** or **Sigil of Were-Rabbit**.

*Sorry, I'm not the best to find names*

**Comment by mywither** [comment]
Personally, "Sigil of Haste" makes a great deal of sense. With there being a "Sigil of the Fast Miner" in the mod, it kind of rules out any sensible possibility of the "Sigil of Haste" positively affecting your mining speed, because that would just be redundant to have two separate items for the same exact purpose.

**Comment by VincentLongiug** [comment]
I agree, with the Sigil of Fast Miner, a Sigil of Haste obviously can't affect Mining speed, but my point is that the Sigil of **Haste** do NOT grant **Haste** Buff, while another Sigil DOES (no matter the actual effect behind the name) and that's where I'm confused.
*If I was insisting about the effects of the Buffs at first, it's because the Haste Buff already exists in Vanilla and, as in BM, increases Mining Speed*

**Comment by mywither** [comment]
And you don't think anyone else realizes that? The devs maybe? If they were going to change it, they would have changed it, being such a tiny detail. I don't think it causes quite as much confusion as you seem to think, in all honesty.

**Comment by mywither** [comment]
I am not the smartest person, especially when it comes to these things. Yet, I have no problem with this, so, I think it's fine.
